wishlist/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles/main.css" />
+    <link
+      rel="stylesheet"
+      href="https://deploy-preview-8--keen-meninsky-a05bc6.netlify.app/home.css"
+    />
+    <link
+      rel="shortcut icon"
+      href="../assets/favicon.ico"
+      type="image/x-icon"
+    />
+    <link
+      rel="stylesheet"
+      href="https://keen-meninsky-a05bc6.netlify.app/home.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://deploy-preview-8--keen-meninsky-a05bc6.netlify.app/styles/utility.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
```

Comment:
It's not clear why are you importing so many sheets. You can make one main and import everything there and use that across.